pay-api/src/pay_api/services/cfs_service.py

```diff
@@ -426,6 +434,33 @@ def add_nsf_adjustment(cls, cfs_account: CfsAccountModel, inv_number: str, amoun
         current_app.logger.debug('>Created CFS Invoice NSF Adjustment')
         return adjustment_response.json()
 
+    @classmethod
+    def adjust_invoice(cls, cfs_account: CfsAccountModel, inv_number: str, amount: float):
+        """Add adjustment to the invoice."""
+        current_app.logger.debug('>Creating Adjustment for Invoice: %s', inv_number)
+        access_token: str = CFSService.get_token().json().get('access_token')
+        cfs_base: str = current_app.config.get('CFS_BASE_URL')
+        adjustment_url = f'{cfs_base}/cfs/parties/{cfs_account.cfs_party}/accs/{cfs_account.cfs_account}/sites/' \
+                         f'{cfs_account.cfs_site}/invs/{inv_number}/adjs/'
+        current_app.logger.debug('Adjustment URL %s', adjustment_url)
+
+        adjustment = dict(
+            comment='Invoice cancellation',
+            lines=[
+                {
+                    'line_number': '1',
+                    'adjustment_amount': str(amount),
```

Comment:
Adjustment amount has to be how much amount needs to be deducted from the invoice. So if invoice amount is 100, then adjustment_amount should be `-100`.

Reply:
Done